package/MDAnalysis/units.py

```diff
@@ -327,6 +328,7 @@
 conversion_factor = {
     'length': lengthUnit_factor,
     'density': densityUnit_factor,
+    'mass': massUnit_factor,
```

Comment:
massUnit_factor is not defined anywhere

Reply:
Yes, there could should be dictionary similar to the other. This would also define how we define our mass. Units could be `kg`, `g` and `u`


```python
massUnit_factor = {
    'u': 1.0,
    'kg': XXX
    'g': XXX,
}
```